packages/x-charts/src/ChartsXAxis/ChartsXAxis.tsx

```diff
@@ -39,10 +39,14 @@ function addLabelDimension(
   {
     tickLabelStyle: style,
     tickLabelInterval,
+    // FIXME: Define the default value in the correct place
+    tickLabelMinGap = 8,
```

Comment:
@JCQuintas @alexfauquette tried adding the default value in `defaultizeAxis`, but `tickLabelMinGap` is still `undefined`. I expected `defaultizeAxis` to be called before an axis is stored in the store instance. Is that expectation incorrect? What would the correct place for creating a default value be?

Reply:
Moved this to `defaultProps` in `ChartsXAxis` for now. Let me know if there's a better place for this 😄